docs/metadata.md

```diff
@@ -65,14 +65,19 @@ needed.
 
 # Metadata image creation
 
-Run `linuxkit run` backends accept a `--data=STRING` option which will
+Run `linuxkit run` backends accept a `-data=STRING` option which will
 cause the given string to be passed to the VM in a platform specific
 manner to be picked up by the `pkg/metadata` component.
 
 Alternatively `linuxkit metadata create meta.iso STRING` will produce
 a correctly formatted ISO image which can be passed to a VM as a CDROM
 device for consumption by the `pkg/metadata` component.
 
+To read `STRING` from a file:
+```
+-data="$(<FILENAME)"
```

Comment:
This is a really odd (and confusing) syntax. could you change this to `-data=<FILENAME>` or something?

Reply:
That's meant to be literal, but I just realized its also a bash-ism.  Would `-data="$(cat FILENAME)"` make it more clear?  Or should I change implementation to actually read a specified file instead of using the command line arguments directly?  I don't know if anyone would need to pass whats usually  multi-line json as a command line argument.

Reply:
no, you can actually pass in a file name instead of a string, though there is an issue with that (see https://github.com/linuxkit/linuxkit/issues/2721)